Flutter/cartesian-charts/axis-customization.md

```diff
@@ -1302,9 +1377,9 @@ In [`DateTimeAxis`](https://pub.dev/documentation/syncfusion_flutter_charts/late
 
 ## RangeController
 
-The [`rangeController`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/ChartAxis/rangeController.html) property is used to set the maximum and minimum values for the chart in the viewport. In the minimum and maximum properties of the axis, you can specify the minimum and maximum values with respect to the entire data source. In the visibleMinimum and visibleMaximum properties, you can specify the values to be viewed in the viewed port i.e. range controller's start and end values respectively.
+The [`rangeController`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/ChartAxis/rangeController.html) property is used to set the maximum and minimum values for the chart in the viewport. In the minimum and maximum properties of the axis, you can specify the minimum and maximum values with respect to the entire data source. In the initialVisibleMinimum and initialVisibleMaximum properties, you can specify the values to be viewed in the viewed port i.e. range controller's start and end values respectively.
```

Comment:
The [rangeController] property defines the lowest and highest data points that are displayed on an axis.

If the same [RangeController] instance is used in both the chart axis and the [RangeSelector], the chart's visible range will be updated based on the range selected in the [RangeSelector].

Reply:
Remove existing RangeController property content details and use my content